File: 1000-1999/1005_Bob_el_jardinero.py
import math

def round2(n):
    return math.floor(n * 100 + 0.5) / 100

T = int(input())

for _ in range(T):
    cadena = input().lower()
    a = cadena.count("a")
    e = cadena.count("e")
    i = cadena.count("i")
    o = cadena.count("o")
    u = cadena.count("u")
    total = len(cadena) - cadena.count(" ")
    print("Caso {0}:".format(_ + 1))
    print("a=", "{0:.2f}".format(round2((a * 100) / total)))
    print("e=", "{0:.2f}".format(round2((e * 100) / total)))
    print("i=", "{0:.2f}".format(round2((i * 100) / total)))
    print("o=", "{0:.2f}".format(round2((o * 100) / total)))
    print("u=", "{0:.2f}".format(round2((u * 100) / total)))

File: 1000-1999/1005_Bob_el_jardinero.cpp
#include <iostream>
#include <iomanip>
using namespace std;

double round2(double n) {
    return (int)(n * 100 + 0.5) / 100.0;
}

int main() {
    int casos;
    cin >> casos;
    cin.ignore();
    for (int n = 0; n < casos; n++) {
        string cadena;
        getline(cin, cadena);
        int a = 0, e = 0, i = 0, o = 0, u = 0;
        for (char c : cadena) {
            switch(c) {
                case 'a':
                    a++;
                    break;
                case 'e':
                    e++;
                    break;
                case 'i':
                    i++;
                    break;
                case 'o':
                    o++;
                    break;
                case 'u':
                    u++;
                    break;
                default:
                    break;
            }
        }
        int total = cadena.size();
        cout << "Caso " << n + 1 << ":" << endl;
        cout << "a= " << fixed << setprecision(2) << round2((a * 100.0) / total) << endl;
        cout << "e= " << fixed << setprecision(2) << round2((e * 100.0) / total) << endl;
        cout << "i= " << fixed << setprecision(2) << round2((i * 100.0) / total) << endl;
        cout << "o= " << fixed << setprecision(2) << round2((o * 100.0) / total) << endl;
        cout << "u= " << fixed << setprecision(2) << round2((u * 100.0) / total) << endl;
    }
    return 0;
}
